Sources/XCHammer/XcodeTargetMap.swift

```diff
@@ -215,5 +206,12 @@ public class XcodeTargetMap {
             return includeTarget(target, pathPredicate: pathsPredicate)
         })
     }()
+    
+    /// Return flattened inner entries
+    public lazy var flattenedInner: Set<XcodeTarget> = Set(allTargets
+        .filter(shouldFlatten)
+        .flatMap { $0.unfilteredDependencies })
+    .filter { !$0.type.hasSuffix("_application") }
```

Comment:
Any special reason for using `_application` and not `application` here?

Reply:
Mostly just to be as specific as possible, and to make it easier to see which flavor of "type" we're looking at. Since there are others like https://github.com/dierksen/xchammer/blob/4b519f30d132975982265799ecc81cf3ac4e7019/Sources/XCHammer/XcodeTarget.swift#L1065-L1066, I just wanted to be as clear as possible what this is supposed to be for future readers.